### PR TITLE
create: Set the hostname to the one provided by docker in the network…

### PR DESCRIPTION
**Pull request by amshinde**
… namespace.

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

File: src/proxy.c [hunk 1]
```diff
@@ -1082,15 +1082,15 @@ cc_proxy_hyper_pod_create (struct cc_oci_config *config)
 	JsonObject                   *ipaddr_obj = NULL;
 	JsonArray                    *ipaddr_arr = NULL;
 
-	if (! (config && config->proxy)) {
+	if (! (config && config->proxy && config->net.hostname)) {
```
Comment (sameo) [inline src/proxy.c]: @amshinde Are we guaranteed to get a networking stanza and a hostname ?

**Comment by chavafg** [comment]
qa-failed

File: src/proxy.c (hunk 1 above)
Comment (amshinde) [inline src/proxy.c]: @sameo Yes, the network setup runs before pod creation. 
https://github.com/01org/cc-oci-runtime/blob/576913626eeefd9d85cd028fe459a9d666f076f5/src/networking.c#L551

**Comment by jcvenegas** [comment]
lgtm

**Comment by sameo** [comment]
LGTM

**Pull request merged**